cargo-typify/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "cargo-typify"
+version = "0.0.11-dev"
+edition = "2021"
+
+[dependencies]
+typify = { version = "0.0.11-dev", path = "../typify" }
+
+clap = { version = "4.1.6", features = ["derive"] }
+color-eyre = "0.6"
```

Comment:
Using `color-eyre` for error handling in the CLI since its the one I tend to use personally, but happy to use a different library if anyone has preferences

https://docs.rs/color-eyre/latest/color_eyre/

Reply:
Neat